OctaneVSPlugin/View/HtmlToFlowDocumentConverter.cs

```diff
@@ -21,7 +21,29 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
             if (value == null)
                 return null;
 
-            var xaml = HtmlToXamlConverter.ConvertHtmlToXaml(value.ToString(), false);
+            try
+            {
+                return ConvertHtmlToFlowDocument(value.ToString());
+            }
+            catch
+            {
+                try
+                {
+                    // if there was an issue trying to convert the original html code
+                    // we show the user an error message
+                    return ConvertHtmlToFlowDocument("<html><body><p><span style=\"color:#e74c3c;\">Unable to obtain information</span></p></body></html>");
+                }
+                catch
+                {
+                    // if we really can't convert to FlowDocument, don't show anything
```

Comment:
is this catch needed, the input is a constant html string

Reply:
I know, I just added it as a backup, in case the converter gets changed and this case isn't tested